base/src/commonMain/kotlin/state/State.kt

```diff
@@ -152,7 +152,7 @@ abstract class StateRepo<S : State, F:MutableSharedFlow<S>>(coroutineContext: Co
     private val initialized = AtomicBoolean(false)
 
     internal open suspend fun initialize(initialValue:S? = null): S =
-        if (initialized.compareAndSet(false, true))
+        if (initialized.compareAndSet(expected = false, new = true))
```

Comment:
just curious, how does this relate to 1.5? did not compile this?

Reply:
not related, just something that was highlighted in IDE

Reply:
better to fix warning in separate PRs unless it's a part of the code you touch, this makes the PR more broad and possibly conflict with other PRs.

I would in fact advice to put this into draft and target `bluetooth2` branch (which also makes changes in this class), then this PR can be merged after that one.

Reply:
reverted